src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -82,6 +82,34 @@ echo "Running '{{ PackageInstallCommand }}'..."
 echo
 {{ PackageInstallCommand }}
 
+{{ if AppInsightsInjectCommand | IsNotBlank }}
+echo
+echo "Preparing environment to set up Application Insights..."
+echo "Running '{{ AppInsightsInjectCommand }}'..."
+echo
+{{ AppInsightsInjectCommand }}
+echo
+echo "Creating Application Insights loader file in '$DESTINATION_DIR'"
+echo "// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+// Created by Oryx
+var appInsights = require('{{ AppInsightsPackageName }}');
```

Comment:
what happens if the package is not available?
I'm thinking that here we should always fail gracefully. For example, if the users run `npm install` by themselves and erase the packages that oryx installed, will we break the app?

Reply:
@gpcastro see my comment above ...